src/wireviz/wv_helper.py

```diff
@@ -53,6 +56,41 @@ def nested_html_table(rows):
     html.append('</table>')
     return html
 
+def html_colorbar(color):
+    return f'<tdX bgcolor="{wv_colors.translate_color(color, "HEX")}" width="4">' if color else None
+
+def html_image(node):
+    if not node.image:
+        return None
+    size = html_size_attr(node.image_size)
+    # The leading attributes belong to the preceeding tag. See where used below.
+    html = f'{"".join(size)}><img scale="{node.image_scale}" src="{node.image}"/>'
+    if size[2]: # fixedsize
+        # Close the preceeding tag and enclose the image cell in a table without
+        # borders to avoid narrow borders when the fixed width < the node width.
+        html = f'''>
+    <table border="0" cellspacing="0" cellborder="0"><tr>
+     <td{html}</td>
+    </tr></table>
+   '''
+    return f'''<tdX{' sides="TLR"' if node.caption else ''}{html}'''
+
+def html_caption(node):
+    return f'''<tdX{' sides="LRB"' if node.image else ''}>{html_line_breaks(node.caption)}''' if node.caption else None
+
+def html_size_attr(size):
+    # Return Graphviz HTML attributes to specify minimum size of a TABLE or TD object
+    # size: List of values where only these are used:
+    # - First value is minimum width of object in points, an int value in the range 1-65535. (Default 0 = none)
+    # - Second value is minimum height of the object in points, an int value in the range 1-65535. (Default 0 = none)
+    # - Third value is a boolean where TRUE specify that the first two values are the fixed image cell size.
+    #   (Default FALSE) Other string values, e.g. FIXED are also interpreted as TRUE.
+    if not size or not isinstance(size, list):
+        return [''] * 3
+    return [f' width="{size[0]}"'  if len(size) > 0 and size[0] else '',
+            f' height="{size[1]}"' if len(size) > 1 and size[1] else '',
+             ' fixedsize="TRUE"'   if len(size) > 2 and size[2] else '']
+
```

Comment:
See comment above.
Just return an empty string (line 89) or do a `"".join()` before returning.

Reply:
I did return a single string in the earlier commit https://github.com/formatc1702/WireViz/pull/153/commits/41b3f3acfedb42dacf600f81985ef67547b58e37, but changed it to avoid the extra nested table when I could avoid it in the next commit. See also my comment to the other code fragment above.

Reply:
With a new `image` class, maybe this can be reversed back to returning a simple string.